docs/src/pages/careers/lead-designer.md

```diff
@@ -0,0 +1,82 @@
+# Lead Designer
+
+<p class="description">We are looking for a lead designer to own most of our design work and manage a small design team.</p>
+
+## Details of the Role
+
+- Location: Remote (preference for UTC-6 to UTC+5).
+- Type of work: Full-time (contractor or employee [depending on circumstances](https://mui-org.notion.site/Hiring-FAQ-64763b756ae44c37b47b081f98915501#494af1f358794028beb4b7697b5d3102))
+- Level: [4 or above](https://docs.google.com/spreadsheets/d/1dDdPD-flNXlgZ0E3ZxVvCDx27RFuhVWJrcfcjNu_I8k/edit#gid=0).
```

Comment:
The link is for Engineers. We need to create one for designers too (unless we already have one).

Reply:
We have one for designers, let's fix the link.